template.distillery/Makefile.mobile

```diff
@@ -291,7 +290,7 @@ $(ADD_PLATFORMS): add-%: $(CORDOVAPATH)/platforms/%
 PLATFORMS := android ios browser
 
 # before building, environment variables are checked and assets are copied.
-$(PLATFORMS): %: check-app-env icons-% assets icons-cp-% add-%
+$(PLATFORMS): %: byte check-app-env icons-% assets icons-cp-% add-%
```

Comment:
@balat What's the purpose of `byte`?
Don't forget to add also it in `emulate-%`.